filebeat/input/docker/config.go

```diff
@@ -35,13 +36,14 @@ type config struct {
 	// Enable CRI flags parsing (to be switched to default in 7.0)
 	CRIFlags bool `config:"cri.parse_flags"`
 
-	// Fore CRI format (don't perform autodetection)
+	// Force CRI format (don't perform autodetection)
 	CRIForce bool `config:"cri.force"`
 }
 
 type containers struct {
-	IDs  []string `config:"ids"`
-	Path string   `config:"path"`
+	IDs    []string `config:"ids"`
+	Path   string   `config:"path"`
+	Suffix string   `config:"suffix"`
```

Comment:
Another option could be to accept glob-based paths here as in the [log input](https://www.elastic.co/guide/en/beats/filebeat/6.5/filebeat-input-log.html#input-paths), this way the default could be `/var/lib/docker/containers/<container_id>/*.log`, and the one for cri-o `/var/log/containers/*-*.log.
I see the combination of path and suffix a bit confussing.